packages/espresso/pubspec.yaml

```diff
@@ -23,4 +23,3 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  pedantic: ^1.10.0
```

Comment:
Opportunistic cleanup; I forgot to remove this when I was cleaning up legacy analysis options.